Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -388,6 +389,12 @@ public final class PackageInfoMapper: PackageInfoMapping {
         for target in resolvedDependencies.keys.sorted() {
             visit(target: target, parentMacOS: false)
         }
+        
+        // TODO: Not hardcode values
+        targets.insert("SwiftRefactor")
+        targets.insert("_SwiftSyntaxTestSupport")
+        targets.insert("SwiftSyntaxMacrosTestSupport")
+        targets.insert("SwiftIDEUtils")
```

Comment:
@waltflanagan / @kwridan here's an interesting use case. A macro, which is in the end a macOS executable, forces downstream dependencies to be macOS, an also the upstream ones. Do you think an upstream traverse makes sense once we've identified which targets are macOS?